### PR TITLE
Compliancies/MinimumRequiredVersion - Invalid value & below minimum supported version

### PR DESCRIPTION
**Pull request by MichielOda**
### New Error Messages

[1.25.9] Minimum required version 'currentMinDmVersion' is lower than the minimum supported version 'currentSupportedDmVersion'.
> Takes in account the MinimumSupportedDataMinerVersion in the ValidatorSettings.

[1.25.10] Invalid value 'tagValue' in tag 'MinimumRequiredVersion'

### CODE REVIEW

File: Protocol/Tests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag.cs [hunk 1]
```diff
@@ -193,7 +199,7 @@ public void CheckUsedFeatures()
 
             if (subResults.Count > 0)
             {
-                IValidationResult minVersionTooLow = Error.MinVersionTooLow(test, null, GetPositionNode(), versionValue, expectedVersion.ToString());
+                IValidationResult minVersionTooLow = Error.MinVersionTooLow(test, null, GetPositionNode(), tag?.RawValue, expectedVersion.ToString());
```
Comment (janstaelensskyline) [inline Protocol/Tests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag.cs:202]: This changed from tag?.value to tag?.rawvalue. Can you verify that was the intention?

File: ProtocolTests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag/Samples/Codefix/BelowMinimumSupportedVersion.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿<Protocol xmlns="http://www.skyline.be/validatorProtocolUnitTest">
+    <Compliancies>
+        <MinimumRequiredVersion>10.1.0.0 - 9966</MinimumRequiredVersion>
```
Comment (janstaelensskyline) [inline ProtocolTests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag/Samples/Codefix/BelowMinimumSupportedVersion.xml:3]: Will this not break when DIS changes what it knows as the minimumRequiredVersion?

Comment (MichielOda) [reply ProtocolTests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag/Samples/Codefix/BelowMinimumSupportedVersion.xml:3]: Current minimum required version is 10.2. This is a default value specified in an internal constructor for the ValidatorSettings which is used for unit tests (also mentioned in xml doc of that constructor).

File: Protocol/Tests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag.cs (hunk 1 above)
Comment (MichielOda) [reply Protocol/Tests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag.cs:202]: Yes as the RawValue is the actual value in the tag itself. This is done to make sure that the user can't be confused by seeing a different value (trimmed, parsed/converted) than what is specified in the tag.

**Review by janstaelensskyline — APPROVED** [review]

**Pull request merged**